.github/workflows/build-binaries.yml

```diff
@@ -60,8 +60,15 @@ jobs:
       - uses: actions/cache@v2
         with:
           path: ./cached_gnat
-          key: ${{ runner.os }}-gnat-ce-2020
+          key: ${{ runner.os }}-gnat-ce-2021
+      - name: Get GNAT Community 2021 toolchain
```

Comment:
Why do you add it here if it wasn't needed before ?

Reply:
We have no Mac OS toolchain in gnat ce 2021. So, for Mac OS it installs gnat-ce-2020 while for other OSs gnat-ce-2021.